streams/src/test/java/org/apache/kafka/streams/processor/internals/StateDirectoryTest.java

```diff
@@ -88,27 +88,6 @@ public void shouldCreateTaskStateDirectory() {
         assertTrue(taskDirectory.isDirectory());
     }
 
-    @Test
-    public void shouldLockTaskStateDirectory() throws IOException {
-        final TaskId taskId = new TaskId(0, 0);
-        final File taskDirectory = directory.directoryForTask(taskId);
-
-        directory.lock(taskId);
-
-        try (
-            final FileChannel channel = FileChannel.open(
-                new File(taskDirectory, StateDirectory.LOCK_FILE_NAME).toPath(),
```

Comment:
Why do we remove this test? Seems, we should update the `FileChannel` here to use the new lock file name?